### PR TITLE
Add discord redirect and remove console.log

### PR DESCRIPTION
**Pull request by ShawnCollinge**
Author: Shawn

## What changes were made?
Adds a discord redirect so we can give people the url http://swecc.org/discord
Also removed a console.log in home.js that someone left.

## Why are these changes important/necessary?
It's nice to be able to share the discord link that way instead of the messy link.

### CODE REVIEW

**Review by elimelt — CHANGES_REQUESTED** [review]

File: src/Components/Redirect.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+function ExternalRedirect({ to }) {
+  useEffect(() => {
+    window.location.href = to;
```
[the remark this answers is not in the document]
Comment (ShawnCollinge) [reply src/Components/Redirect.js:5]: This is for directly linking, like I give you the url http://swecc.org/discord I'm not sure why you'd want 2 tabs to open then? For footer links etc you add the _blank to that.

File: src/Components/Footer.js [hunk 2]
```diff
@@ -21,17 +21,17 @@ const Footer = () => {
...
-            <a href="https://instagram.com/swecc.uw?igshid=YmMyMTA2M2Y=" target="_blank" rel="noopener noreferrer">
+            <a href="/instagram" target="_blank" rel="noopener noreferrer">
               <img className="contact-images" src={instaLogo} alt="instalogo" />
             </a>
-            <a href="https://discord.com/invite/Pbk4sCEWDY" target="_blank" rel="noopener noreferrer">
+            <a href="/discord" target="_blank" rel="noopener noreferrer">
               <img className="contact-images" src={discordLogo} alt="discord" />
             </a>
-            <a href="https://www.linkedin.com/company/software-engineering-career-club-at-uw/" target="_blank" rel="noopener noreferrer">
+            <a href="/linkedin" target="_blank" rel="noopener noreferrer">
```
Comment (ShawnCollinge) [inline src/Components/Footer.js:30]: @elimelt the blank goes here. The url just is a redirect that we can link on other websites or on flyers etc.

File: src/Components/Redirect.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (elimelt) [reply src/Components/Redirect.js:5]: ohhhh wait sorry, I just read External and assumed it meant to a different site

File: src/Components/Footer.js (hunk 2 above)
Comment (elimelt) [reply src/Components/Footer.js:30]: Even for event handlers, anything that navigates to a different page should use _blank imo

**Review by elimelt — APPROVED** [review]
LG, TM

**Pull request merged**